nemo_skills/pipeline/utils.py

```diff
@@ -131,6 +143,82 @@ def check_if_mounted(cluster_config, path_to_check):
         raise ValueError(f"The path '{path_to_check}' is not mounted. Check cluster config.")
 
 
+def check_remote_mounts(
```

Comment:
Better to name it just check_mounts as this is also applicable when running locally and specifying config file

Reply:
Done